Infusion.LegacyApi/Targeting.cs

```diff
@@ -233,6 +233,19 @@ public void TargetTile(Location3D location, ModelId tileType)
             client.TargetLocation(lastCursorId, location, tileType, CursorType.Cancel);
         }
 
+        public void TargetTile(Location2D location)
+        {
+            var tiles = Ultima.Map.Felucca.Tiles.GetStaticTiles(location.X, location.Y);
```

Comment:
All code in Targeting is very fragile and I wouldn't touch it or add anything to it without adding related tests. Also adding dependency to Ultima.Map on this level is not appropriate. Retrieving a tile from specific location can be moved to Legacy class.